packages/pages/src/dev/server/middleware/serveHttpFunction.ts

```diff
@@ -15,11 +15,14 @@ export const serveHttpFunction = async (
   const argument: HttpFunctionArgument = {
     queryParams: req.query as { [p: string]: string },
     pathParams: req.params,
+    method: req.method,
```

Comment:
Doesn't really matter, but it's method header body here where elsewhere its headers method body